src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ButtonHTMLAttributes, MouseEventHandler, ReactNode } from 'react';
+
+// Styles
+import './button.css';
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: string;
```

Comment:
variant should type: primary, secondary, tertiary

Reply:
Done